index.html

```diff
@@ -135,5 +135,24 @@ <h2>About <br>Myself</h2>
             <li class="about-skill-list border">Skills <img src="./img/chevron-right.png" alt="chevron" class="chevron"></li>
         </ul>
     </section>
+    <!-- Contact me section-->
+    <section class="contact-form">
+        <div class="background-img">
+            <!-- Adding background image through css -->
+        </div>
+        <h2 class="contact-me">Contact me</h2>
+        <!--support message-->
+        <p class="support-msg">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+        <form action="https://formspree.io/f/mjvzkgvn" method="POST" class="form">
+            <input type="text" name="user_name" placeholder="Florin" maxlength="30" required>
+            <input type="email" name="user_email" placeholder="vflorin1995@gmail.com" required>
+            <textarea name="user_comment" class="comment" maxlength="500" required placeholder="Write your message here."></textarea>
+            <button type="submit" class="contact-btn">
+                Get it touch
+            </button>
```

Comment:
- [ ] Kindly add button interactions such as hover, and active (when it's pressed) as shown in the Figma template.